src/app/public/src/modules/clipboard/clipboard.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="clipboard-wrapper" stacheClipboard clipboardTrigger="{{clipboardTrigger}}">
+  <ng-content></ng-content>
+  <button
+    *ngIf="chooseTrigger()"
+    class="sky-btn sky-btn-default clipboard-btn"
+    id="clipboardButton">
```

Comment:
Avoid using `id` within a reusable Angular component, since there could be multiple elements on the page with the same ID.